docs/guides/platform/object-storage/using-object-storage-with-akamai-cdn/index.md

```diff
@@ -0,0 +1,101 @@
+
+-   **Rate limits:** Object storage solutions often have rate limits imposed on a requests per second (RPS) basis. If users are accessing Object Storage endpoints directly without the use of a CDN, there is a greater chance RPS limits are reached as more users access content.
+
+-   **Updating manifests:** Bucket contents can be determined by syncing with a manifest. This means that when source manifests are updated often, buckets may constantly have their contents changed.
+
+### Internet Considerations
+
+There are multiple internet-related factors to consider when building out a content delivery solution:
+
+-   **First mile:** The content provider connection point. Should content providers set up content to be accessible from a single physical location (i.e. an origin data center), user access is limited by “first mile” connectivity limitations.
+
+-   **Last mile:** The end-user connection point. Slow user internet connection speeds are uncontrollable and unpredictable. They can also mask or obfuscate problems related to the “first mile”, network peering, and bottleneck issues.
+
+-   **Peering points:** The connection points between two networks. These connections are not guaranteed and are difficult to troubleshoot. Networks benefit financially from connectivity with content providers and end users but not each other.
+
+-   **Backbone:** The infrastructure and physical connections that make up the internet. Hardware and network capacity limitations can cause performance issues.
+
+### Data Center-Based Architectures
+
+Consider the data center-based architecture below. Without the use of a CDN, object storage content distribution is limited to individual, origin data centers. Data center-based delivery relies heavily on public networks to pull and deliver content from regional origin points. At scale, this may result in bandwidth issues, high latency, and inefficient experiences for large numbers of users.
+
+![DC-Based Architecture](DC-Based-Architecture.png)
+
+## Advantages of Using Akamai CDN
+
+CDNs are crucial for data delivery solutions because they cache content like images, videos, and other files on edge servers closer to end-users. This reduces latency and overall load on origin servers, improves delivery speed and efficiency, and saves money by reducing origin requests and lowering egress. For example, in use cases such as content streaming, CDNs can cache segments from streams to reduce startup times, limit stream interruptions, and eliminate potential buffering issues.
+
+Object storage can’t scale linearly with the number of users, but using it as an origin point for CDN delivery overcomes this by allowing users to interact with the CDN instead of the object storage system. Applications can then store objects independently of user traffic, with the added benefit of CDN-based security. Using Linode Object Storage on Akamai Connected Cloud with Akamai CDN can also significantly reduce object storage-based egress costs.
```

Comment:
it feels like "Applications can then store objects" should be "Applications can then serve objects"

Reply:
edited